final-assignment-tests/shell/test-037.sh

```diff
@@ -0,0 +1,20 @@
+#/bin/bash
+
+dir=long-path-123456
+tmpdir=$(mktemp -d)
```

Comment:
It would be prudent to check if the `tmpdir` is non-empty and `exit 1` otherwise.